### ISSUE TITLE
Description not available - Portable Application

### ISSUE THREAD

**Issue by hshah220** [LluisV/Z-Anatomy#10]
The window to view the description when selecting a bone is not working in the portable application file. I have attached a screenshot to demonstrate. I was wondering what needs to be changed and if anyone can guide me to resources to ammend this.
![image](https://github.com/LluisV/Z-Anatomy/assets/47408739/69db9940-e110-4045-b40e-bb4790bea31f)

**Comment by hshah220** [comment]
Hi, I was wondering if there is an update on this issue, or suggestions how it could be fixed?

**Comment by LluisV** [comment]
Hi, this is a known issue in the WebGL version. It seems like a bug in Unity itself. Does this happen in the standalone Windows version too?

**Comment by hshah220** [comment]
Yes, this issue occurs in the Windows standalone version as well. I built a version on WebGL using the application files on Github, and the issue occurs in that version as well. I tried the mobile version from github, and the issue occurs on that as well.

**Comment by rehandaphedar** [comment]
On [the website](https://z-anatomy.com), changing the language to anything other than English makes the description appear. On changing the language back to English, it still works. Not sure how this fixes it or what is causing the issue in the first place.
